deepchem/feat/graph_features.py

```diff
@@ -668,6 +671,10 @@ def __init__(self, master_atom=False, use_chirality=False,
       provided in atom_properties.  So "atom 00000000 sasa" would be the
       name of the molecule level property in mol where the solvent
       accessible surface area of atom 0 would be stored.
+    per_atom_fragmentation: Boolean
```

Comment:
Add a brief description here of when atom-derived featurizations would be useful to use